### PR TITLE
fix: table-prefix-cls is not work in class 'ant-table-thead'

### PR DESCRIPTION
**Pull request by jerikchan**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch. Pull request will be merged after one of collaborators approve. Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/vueComponent/ant-design-vue/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

> 1. Describe the source of requirement.
> 2. Resolve what problem.
> 3. Related issue link.
#2165 

### API Realization (Optional if not new feature)

> 1. Basic thought of solution and other optional proposal.
> 2. List final API realization and usage sample.
> 3. GIF or snapshot should be provided if includes UI/interactive modification.

### What's the effect? (Optional if not new feature)

> 1. Does this PR affect user? Which part will be affected?
> 2. What will say in changelog?
> 3. Does this PR contains potential break change or other risk?

### Changelog description (Optional if not new feature)

> 1. English description
> 2. Chinese description (optional)

### Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by jerikchan** [comment]
@tangjinzhou 为啥编译没有通过？

File: components/table/style/index.less [hunk 1]
```diff
@@ -740,7 +740,7 @@
       color: transparent;
       pointer-events: none;
     }
-    .ant-table-thead > tr > th:first-child {
+    .@{table-prefix-cls} > tr > th:first-child {
```
Comment (tangjinzhou) [inline components/table/style/index.less:743]: .@{table-prefix-cls}-thead   ？

**Comment by jerikchan** [comment]
fixed，但编译还没有通过

**Comment by tangjinzhou** [comment]
编译问题忽略

**Pull request merged**